.circleci/config.yml

```diff
@@ -50,10 +42,17 @@ jobs:
           name: Cleanup
           command: rm -rf ./dependency-check
 
-  lint-and-test: &lint-and-test
-    executor: node-container
+  lint-and-test:
+    docker:
+       - image: node:15
```

Comment:
Node 15 is not LTS, it should not be the base

Reply:
Agree - this was leftover when testing around with installing different npm versions